README.md

```diff
@@ -1,16 +1,17 @@
 # Python Library
 
 ![Build](https://github.com/8percent/python-library/actions/workflows/ci.yml/badge.svg)
+[![codecov](https://codecov.io/gh/8percent/python-library/branch/master/graph/badge.svg?token=J7S8RQ32Y0)](https://codecov.io/gh/8percent/python-library)
```

Comment:
This line should not be added. since we no longer uses Codecov as our coverage tool.

Codecov removed in [here](https://github.com/8percent/python-library/commit/ce27caa47350b4246b6bd5be631c6f5b137889f0)

Reply:
Good point! I will make the necessary changes and update accordingly.